pkg/apis/rollouts/v1alpha1/analysis_types.go

```diff
@@ -547,6 +547,8 @@ type KayentaMetric struct {
 	Threshold KayentaThreshold `json:"threshold" protobuf:"bytes,7,opt,name=threshold"`
 
 	Scopes []KayentaScope `json:"scopes" protobuf:"bytes,8,rep,name=scopes"`
+
+	Lookback bool `json:"lookback,omitempty" protobuf:"varint,9,opt,name=lookback"`
```

Comment:
I'm not sure why `make codegen` did not regenerate the `generated.proto` despite introducing this new field. Something doesn't seem to be working properly.